code/modules/mob/living/bot/medbot.dm

```diff
@@ -84,7 +84,7 @@
 		return
 
 	if(H.stat == DEAD)
-		var/death_message = pick("What a shame.", "What a rotten way to die.", "They were a good man.")
+		var/death_message = pick("What a shame.", "What a rotten way to die.")
```

Comment:
```suggestion
		var/death_message = pick("What a shame.", "What a rotten way to die.", "They were a good man.")
```

Reply:
Why revert the Deus Ex reference that was added by our server, only to our server, and, within the last week?